scripts/ci-test.sh

```diff
@@ -43,7 +43,7 @@ sqlitetest() {
 oracletest() {
   # Install Oracle
   docker pull wnameless/oracle-xe-11g
-  docker run -d -p 49160:22 -p 49161:1521 wnameless/oracle-xe-11g
+  docker run -d -p 49160:22 -p 1521:1521 wnameless/oracle-xe-11g
```

Comment:
The port was changed in Alan's pull request to ripple-ilp-server and we should keep it consistent.